implementations/rust/message/src/lib.rs

```diff
@@ -0,0 +1,540 @@
+#![allow(unused)]
+
+// Definition and implementation of an Ockam message and message components.
+// Each message component, and the message overall, implements the "Codec" trait
+// allowing it to be encoded/decoded for transmission over a transport.
+
+pub mod message {
```

Comment:
I would move this to a file named `message.rs`. Then you can dispense with the pub mod group

Reply:
done